app/webpacker/components/Results/api/records.js

```diff
@@ -0,0 +1,8 @@
+import { fetchJsonOrError } from '../../../lib/requests/fetchWithAuthenticityToken';
+import { recordsUrl } from '../../../lib/requests/routes.js.erb';
+
+// eslint-disable-next-line import/prefer-default-export
+export async function getRecords(eventId, region, gender, show) {
+  const { data } = await fetchJsonOrError(recordsUrl(eventId, region, gender, show), { headers: { Accept: 'application/json' } });
```

Comment:
Is the manual header necessary if you're using `fetchJsonOrError`?

Reply:
fetch fetchJsonOrError does not set any headers itself